manager/controllers/app/policy_compiler_interface.go

```diff
@@ -86,12 +84,12 @@ func LookupPolicyDecisions(datasetID string, policyManager connectors.PolicyMana
 
 	err = ValidatePolicyDecisionsResponse(openapiResp, PolicyManagerTaxonomy)
 	if err != nil {
-		log.Print("Error after calling ValidatePolicyDecisionsResponse:", err)
+		appContext.Log.Error().Err(err).Msg("error while validating policy manager response")
 		return actions, errors.New("Validation error: " + err.Error())
 	}
 
 	output = render.AsCode(openapiResp)
-	log.Print("openapi response received from policy manager: ", output)
+	appContext.Log.Info().Msgf("response from policy manager: %s", output)
```

Comment:
Please add the datasetid and uuid

Reply:
Added datasetid, uuid is already inside.